packages/scene-composer/.eslintrc.js

```diff
@@ -72,5 +72,5 @@ module.exports = {
     jest: true,
     es6: true,
   },
-  ignorePatterns: ['src/three/GLTFLoader.js', 'src/three/tiles3d/*'],
+  ignorePatterns: ['src/three/GLTFLoader.js', 'src/three/tiles3d/*', 'tools/watch-build.js'],
```

Comment:
Did you look at just running the watch command with typescript instead of nodemon? This is a built in feature:

https://www.typescriptlang.org/docs/handbook/configuring-watch.html

Reply:
nodemon was the first showing when I'm searching for watch files, so I stated with it.

I just tried this tsc watch feature, it works fine too. but since our build script includes all those steps `"run-s extract-msgs convert-svg tttsc copy-assets"` it won't be able to auto build for svg file changes for example. 

I could add both commands and let people choose which one to use - either faster but only for ts file changes, or slower but all changes

Reply:
This is just rerunning the entire build for every change, that's very heavy. This is ok for now, but we should look for a better option. Typically these sort of scripts aren't needed.